core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -1,6 +1,8 @@
 package cats
 package data
 
+import functor.Bifunctor
```

Comment:
Nitpicking, but given we seem to use the fully qualified name in other imports, maybe `import cats.functor.Bifunctor`